astropy/units/equivalencies.py

```diff
@@ -811,3 +811,31 @@ def with_H0(H0=None):
     h100_val_unit = Unit(100/(H0.to_value((si.km/si.s)/astrophys.Mpc)) * astrophys.littleh)
 
     return Equivalency([(h100_val_unit, None)], "with_H0", kwargs={"H0": H0})
+
+
+def cosmological(z, cosmology=None, comoving=False):
```

Comment:
I actually wouldn't call this ``cosmological`` as there are other important equivalencies in cosmology, like temperature and redshift. Maybe (and I'm not thrilled with this long name either) 

```suggestion
def cosmological_angle_distance(z, cosmology=None, comoving=False):
```

Reply:
@mhvk, would you consider ``cosmo`` to be the Astropy de facto standard shortening for ``cosmology``? I'm just trying to decide whether to recommend 

```suggestion
def cosmological(z, cosmo=None, comoving=False):
```

Reply:
I wouldn't abbreviate it.

Reply:
Now ``astropy/cosmology`` has its own units module for both units and equivalencies.
This equivalency should be moved to there.